routes/project.js

```diff
@@ -31,7 +33,16 @@ module.exports = {
  * ]
  */
 function getProjects(req, res, next) {
-  Project.findAll()
+  let search = {};
+
+  //FIXME: this is probably not the best way to implement this
+  // since a search for a name should always return a maximum of 1 item
```

Comment:
There may be function that's a happy medium (does `.find` exist?) but I agree that this works for now and we can always revisit later.